src/internal_events/kubernetes/reflector.rs

```diff
@@ -10,11 +10,11 @@ pub struct InvocationDesyncReceived<E> {
 
 impl<E: std::fmt::Debug> InternalEvent for InvocationDesyncReceived<E> {
     fn emit_logs(&self) {
-        warn!(message = "Handling desync.", error = ?self.error);
+        warn!(message = "Handling invocation desync.", reason = ?self.error);
```

Comment:
Could we update these events to match the new component spec description for "error events"? 

https://github.com/vectordotdev/vector/blob/master/docs/specs/component.md#error

I realize these are warnings, but I think the spirit is the same.

Reply:
@jszwedko @bruceg should the spec expand Error to include Warn specifically?

Reply:
Not to be difficult, but I don't believe I could reasonably match the defined spec from these particular events

Reply:
Yeah, I think it would be good to call out warnings separate from errors in the component spec. Would you want to update that?

That being said, this isn't blocking since we do have another issue to survey the instrumentation of all components.

Reply:
> Not to be difficult, but I don't believe I could reasonably match the defined spec from these particular events

That could mean the spec needs to be updated then 😄

Reply:
Perhaps but I'd consider those to be "large" bodies of work, and out of scope from here. These events are not related to the dropping/aborting/flow of events which the spec seems fairly focused in at this point